src/NuGet.Core/NuGet.Frameworks/NuGetFrameworkFactory.cs

```diff
@@ -73,9 +73,9 @@ public static NuGetFramework Parse(string folderName, IFrameworkNameProvider map
         /// <summary>
         /// Creates a NuGetFramework from individual components
         /// </summary>
-        public static NuGetFramework ParseComponents(string targetFrameworkIdentifier, string targetFrameworkVersion, string targetFrameworkProfile, string targetPlatformIdentifier, string targetPlatformVersion)
+        public static NuGetFramework ParseComponents(string targetFrameworkMoniker, string targetPlatformMoniker)
```

Comment:
What happened to MinVersion?

Reply:
It was never used in the framework parsing.